.gitignore

```diff
@@ -2,7 +2,9 @@ node_modules/
 
 # Gatsby
 .cache/
-public/
+
+# Gatsby + production
+public/ 
```

Comment:
Why separate this from the Gatsby section?

Reply:
I wanted to add further clarification for what it was for.